src/Select.js

```diff
@@ -118,10 +118,10 @@ var Select = React.createClass({
 				return;
 			}
 			var menuElem = React.findDOMNode(this.refs.selectMenuContainer);
-			var controlElem = React.findDOMNode(this.refs.control);
+			var wrapperElem = React.findDOMNode(this.refs.control);
 
 			var eventOccuredOutsideMenu = this.clickedOutsideElement(menuElem, event);
-			var eventOccuredOutsideControl = this.clickedOutsideElement(controlElem, event);
+			var eventOccuredOutsideControl = this.clickedOutsideElement(wrapperElem, event);
 
 			// Hide dropdown menu if click occurred outside of menu
 			if (eventOccuredOutsideMenu && eventOccuredOutsideControl) {
```

Comment:
In this case you probably don't need to check both  for `eventOccuredOutsideMenu && eventOccuredOutsideControl` cause both `selectMenuContainer ` and `control` are children of `wrapper`. This should be enough:

`if (eventOccuredOutsideControl) {`